e2e/ansible/playbooks/hyperconverged/test-replica-node-affinity/replica-node-affinity.yml

```diff
@@ -61,35 +59,35 @@
            replace: 'openebs.io/jiva-replica-count: "2"'
```

Comment:
Please check the comment above to prevent hardcoding the replica count

Reply:
Removed